torch_np/_funcs_impl.py

```diff
@@ -913,6 +913,28 @@ def take_along_axis(arr: ArrayLike, indices: ArrayLike, axis):
     return torch.take_along_dim(arr, indices, axis)
 
 
+def put(a: ArrayLike, ind: Sequence[ArrayLike], v: ArrayLike, mode="raise"):
+    if mode != "raise":
+        raise NotImplementedError(f"{mode=}")
```

Comment:
This can probably be implemented via https://github.com/Quansight-Labs/numpy_pytorch_interop/pull/114